src/conversations/Conversation.ts

```diff
@@ -27,9 +27,29 @@ import {
   PublicKeyBundle,
   SignedPublicKeyBundle,
 } from '../crypto'
+import { PreparedMessage } from '../PreparedMessage'
 import { sha256 } from '../crypto/encryption'
-import { ContentTypeText } from '../codecs/Text'
 import { buildDecryptV1Request, getResultOrThrow } from '../utils/keystore'
+import { ContentTypeText } from '../codecs/Text'
+
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+
+type ConversationV1Export = {
```

Comment:
This must be a weird merge conflict. These types are removed in beta/v8

Reply:
Removed in https://github.com/xmtp/xmtp-js/pull/321/commits/7b1131f2c57f2d8347c59e16f9723400e1d00490.